lib/cocoapods-core/source.rb

```diff
@@ -326,6 +369,18 @@ def specs_dir
       end
     end
 
+    def git_commit_hash
+      (`git rev-parse HEAD` || '').strip
+    end
+
+    def update_git_repo
```

Comment:
No, the output here will no longer be streamed -- it will only be printed after the command is finished.